File: Car-Gears/gears.java
class Car {

    public int gear;
    public int speed;

    public Car(int gear , int speed)
    {
        this.gear = gear;
        this.speed = speed;
    }

    public void applyBrake(int decrement)
    {
        speed -= decrement;
    }

    public void speedUp(int increment)
    {
        speed += increment;
    }
    public String toString()
    {
            return("No. of gears are : " + gear + "\n" + "Speed of SuperCar is : " + speed + " mph");
    }
}

class SuperCar extends Car {

    public int setHeight;
    public SuperCar(int gear, int speed, int rideHeight)

    {
        super(gear, speed);
        setHeight = rideHeight;
    }

    public void setHeight(int newValue)
    {
        setHeight = newValue;
    }
    @Override public String toString()
    {
        return (super.toString() + "\nRide height is : " + setHeight + " inch");
    }
}
public class gears {
    public static void main(String[] args) {
        SuperCar mb = new SuperCar(5, 180, 15);
        System.out.println(mb.toString());
    }
}

File: Car-Gears/gears.kt
internal open class Car(var gear: Int, var speed: Int) {
    fun applyBrake(decrement: Int) {
        speed -= decrement
    }

    fun speedUp(increment: Int) {
        speed += increment
    }

    override fun toString(): String {
        return "No. of gears are : $gear\nSpeed of SuperCar is : $speed mph"
    }
}

internal class SuperCar(gear: Int, speed: Int, var setHeight: Int) : Car(gear, speed) {
    fun setHeight(newValue: Int) {
        setHeight = newValue
    }

    override fun toString(): String {
        return """${super.toString()}
Ride height is : $setHeight inch"""
    }
}

object gears {
    @JvmStatic
    fun main(args: Array<String>) {
        val mb = SuperCar(5, 180, 15)
        println(mb.toString())
    }
}